header/pb/extended_header.proto

```diff
@@ -4,13 +4,13 @@ package header.pb;
 
 import "tendermint/types/types.proto";
 import "tendermint/types/validator.proto";
-import "tendermint/da/data_availability_header.proto";
+import "da/data_availability_header.proto";
 
 message ExtendedHeader {
   tendermint.types.Header header = 1;
   tendermint.types.Commit commit = 2;
   tendermint.types.ValidatorSet validator_set = 3;
-  tendermint.da.DataAvailabilityHeader dah = 4;
```

Comment:
[for reviewers] this type moved from celestia-core to celestia-app in https://github.com/celestiaorg/celestia-app/pull/706